src/js/outreach/apps/verify_app.js

```diff
@@ -71,12 +73,9 @@ export default App.extend({
           this.stop({ isVerified: true });
         })
         .catch(response => {
-          if (response.status === 403) {
-            this.showVerifyCodeView(true);
-            return;
-          }
+          if (response.status >= 500) return;
 
-          this.showDialogErrorView();
+          this.showVerifyCodeView(true);
         });
```

Comment:
This is request either returns `201` (success), `403`, or a `500` server error.

If `500`, we’ll bail from this `.catch()` and have the error app handle things. Otherwise, we can assume it was the `403` incorrect code error.